app/styles/modules/_settings.scss

```diff
@@ -200,6 +200,12 @@ body.settings #main-content.card {
         }
       }
 
+      html[dir] .clients & {
```

Comment:
Because everyone loves a specificity war.

Reply:
This made me laugh a little more than it should have.